src/js/tabs/receiver.js

```diff
@@ -90,6 +90,7 @@ TABS.receiver = {
         { name: 'None',                 id: 0,   feature: null,           telemetry: 0,           visible: true, },
         { name: 'TBS CRSF',             id: 9,   feature: 'RX_SERIAL',    telemetry: 0x0010378F,  visible: true, },
         { name: 'Futaba S.BUS',         id: 2,   feature: 'RX_SERIAL',    telemetry: 0,           visible: true, },
+        { name: 'Futaba S.BUS2',        id: 15,  feature: 'RX_SERIAL',    telemetry: 0xFFFFFFFF,  visible: true, },
```

Comment:
It looks like the telemetry sensors are not configurable, right?
If so, the configurable sensor field should be `telemetry: 0`

Reply:
Changed to 0.